### PR TITLE
Feature stream

### PR DESCRIPTION
**Pull request by gllb**
$ make testacc TEST='./flexibleengine' TESTARGS='-run TestAccDis*'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./flexibleengine -v -run TestAccDis* -timeout 720m
=== RUN   TestAccDisStreamV2_basic
=== PAUSE TestAccDisStreamV2_basic
=== CONT  TestAccDisStreamV2_basic
--- PASS: TestAccDisStreamV2_basic (8.97s)
PASS
ok      github.com/terraform-providers/terraform-provider-flexibleengine/flexibleengine 9.044s

### CODE REVIEW

File: flexibleengine/resource_flexibleengine_dis_stream_v2.go [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+	return &schema.Resource{
+		Create: resourceDisStreamCreate,
+		Read:   resourceDisStreamRead,
+		Update: resourceDisStreamUpdate,
+		Delete: resourceDisStreamDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"stream_name": {
```
Comment (ShiChangkuo) [inline flexibleengine/resource_flexibleengine_dis_stream_v2.go:25]: suggest to use **name** instead

File: flexibleengine/resource_flexibleengine_dis_stream_v2.go [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+				ValidateFunc: validation.StringMatch(regexpStreamName,
+					"1 to 64 in length, only letters, digits, hyphens (-), and underscores (_) are allowed."),
+			},
+			"stream_type": {
```
Comment (ShiChangkuo) [inline flexibleengine/resource_flexibleengine_dis_stream_v2.go:32]: suggest to use **type** instead

File: flexibleengine/resource_flexibleengine_dis_stream_v2.go [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				Default:      streams.StreamTypeCommon,
+				ValidateFunc: validation.StringInSlice([]string{streams.StreamTypeCommon, streams.StreamTypeAdvanced}, false),
+			},
+			"partition_count": {
+				Type:     schema.TypeInt,
+				Required: true,
+				// ForceNew: true, TODO: Check if update do the job
```
Comment (ShiChangkuo) [inline flexibleengine/resource_flexibleengine_dis_stream_v2.go:42]: seems that this parameter can not be updated in OTC

File: flexibleengine/resource_flexibleengine_dis_stream_v2.go [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+	return nil
+}
+
+func resourceDisStreamUpdate(d *schema.ResourceData, meta interface{}) error {
+	return resourceDisStreamRead(d, meta)
+}
+
+// func validatePartitionCount(val interface{}, key string) (warns []string, err []error) {
+// }
```
Comment (ShiChangkuo) [inline flexibleengine/resource_flexibleengine_dis_stream_v2.go:143]: please clean up unused codes.

File: flexibleengine/resource_flexibleengine_dis_stream_v2.go (hunk 1 above)
Comment (gllb) [reply flexibleengine/resource_flexibleengine_dis_stream_v2.go:25]: done

File: flexibleengine/resource_flexibleengine_dis_stream_v2.go (hunk 2 above)
Comment (gllb) [reply flexibleengine/resource_flexibleengine_dis_stream_v2.go:32]: done

File: flexibleengine/resource_flexibleengine_dis_stream_v2.go (hunk 3 above)
Comment (gllb) [reply flexibleengine/resource_flexibleengine_dis_stream_v2.go:42]: done

File: flexibleengine/resource_flexibleengine_dis_stream_v2.go (hunk 4 above)
Comment (gllb) [reply flexibleengine/resource_flexibleengine_dis_stream_v2.go:143]: done

**Pull request merged**